cc/README.md

```diff
@@ -98,6 +98,54 @@ fairly simple (there is no need for a LibertyTracker) and at the time of writing
 performance of the Position code was more than 450x that of its Python
 counterpart.
 
+## Inference engines
+
+C++ Minigo currently supports three separate engines for performing inference:
+
+ - tf: peforms inference using the TensorFlow libraries built by
+   `cc/configure_tensorflow.sh`.
+ - remote: launches a Python subprocess that performs inference using the
+   version of TensorFlow installed on the system. The remote inference
```

Comment:
maybe mention this this version of tensorflow found on the current PYTHONPATH ?

Reply:
I don't think it loads off PYTHONPATH, mine doesn't actually contain anything useful. I think it uses PATH

Reply:
but it will launch whatever `python` is active for the environment you launch minigo in, though, correct?  Like if your venv is activated, its in that venv python, etc.

Reply:
Yes. I've updated the doc to explicitly mention that